lib/geoblacklight/dataingest.rb

```diff
@@ -16,14 +16,12 @@ class DataIngest
     "dc:creator": {required: true},
     "dc:language": {required: true},
     "dc:publisher": {required: true},
-    "dc:relation": {required: false},
     "dc:type": {required: true},
     "dct:spatial": {required: true},
     "dct:temporal": {required: false},
     "dct:issued": {required: false},
     "ispartof": {required: true},
     "georss:box": {required: true},
```

Comment:
I saw you made changes on line 76. But will this "georss:box" field be completely replaced by "solr_gem"?

Reply:
Is "dc:type" optional now?